pkg/acsengine/defaults-apiserver.go

```diff
@@ -17,7 +17,7 @@ func setAPIServerConfig(cs *api.ContainerService) {
 		"--audit-log-maxage":           "30",
 		"--audit-log-maxbackup":        "10",
 		"--audit-log-maxsize":          "100",
-		"--audit-log-path":             "/var/log/apiserver/audit.log",
+		"--audit-log-path":             "/var/log/audit.log",
```

Comment:
@pidah what's the reason for this change?

Reply:
@jackfrancis /var/log/apiserver directory does not exist

Reply:
oy